Formula/markdown-toc.rb

```diff
@@ -0,0 +1,25 @@
+require "language/node"
+
+class MarkdownToc < Formula
+  desc "Generate a markdown TOC (table of contents) with Remarkable"
+  homepage "https://github.com/jonschlinkert/markdown-toc"
+  url "https://registry.npmjs.org/markdown-toc/-/markdown-toc-1.2.0.tgz"
+  sha256 "4a5bf3efafb21217889ab240caacd795a1101bfbe07cd8abb228cc44937acd9c"
+  license "MIT"
+
+  livecheck do
+    url :stable
+  end
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_equal "- [One](#one)\n- [Two](#two)",
+      shell_output("bash -c \"#{bin}/markdown-toc - <<< $'# One\\n\\n# Two'\"").strip
```

Comment:
Doesn't `pipe_output` work here?

Reply:
I'm not really sure how the tests work, I based my test by looking at another formula